packages/scm/src/browser/scm-provider.ts

```diff
@@ -58,6 +58,8 @@ export interface ScmResource {
 }
 
 export interface ScmResourceDecorations {
+    icon?: string;
```

Comment:
Not sure whether it would be a better idea to keep the conditional structure (either a ThemeIcon or a { dark: ..., light...} structure until we render the whole thing in the UI? It kinda doesn't make sense to have a dark and light ThemeIcon, as mentioned elsewhere, IMO.